File: Codeforces/Div3/786/Number-Transformation/main.kt
// https://codeforces.com/contest/1674/problem/A

fun readLn() = readLine()!!
fun readInt() = readLn().toInt()
fun readInts() = readLn().split(" ").map {it.toInt()}

fun main () {
    for (i in 1..readInt()) {
        val (x, y) = readInts()

        if (x > y || y % x > 0) {
            println("0 0")
            continue
        }

        println(1.toString().plus(" ").plus(((y/x).toInt().toString())))
    }
}

File: Codeforces/Div3/786/Number-Transformation/main.cpp
// https://codeforces.com/contest/1674/problem/A

#include<bits/stdc++.h>

using namespace std;

int main () {
    cin.tie(0);
    ios_base::sync_with_stdio(0);
    int t; cin >> t;
    while(t--) {
        int x, y; cin >> x >> y ;
        
        if (x > y || y % x) {
            cout << 0 << " " << 0 << endl;
            continue;
        }

        cout << 1 << " " << (int)(y/x) << endl;
    }
    return 0;
}
